File: atvd presenças/1.py
nome = input("Digite o seu nome: ")
sl = float(input("Digite o salário atual: "))
ida = 0
if 0 < sl <= 500:
    ida = sl * 0.15
elif 500 < sl <= 700: 
    ida = sl * 0.12
elif 700 < sl <= 1000:
    ida = sl * 0.10
elif 1000 < sl <= 1800:
    ida = sl * 0.07
elif 1800 < sl <= 2500:
    ida = sl * 0.04
elif sl > 2500:
    ida = 0
else:
    print("Digite um valor válido!")
    exit() 
nvs = sl + ida
percentual_aumento = (ida / sl) * 100
print("Nome do usuário:", nome)
print("Percentual de aumento: %.2f%%" % percentual_aumento)
print("Salário atual: R$%.2f" % sl)
print("Novo salário: R$%.2f" % nvs)

File: atvd presenças/1.c
#include <stdio.h>
#include <stdlib.h>
int main() {
    char nome[61];
    float sl, ida = 0, nvs = 0;
    printf("Digite o seu nome: ");
    scanf("%s", nome); 
    printf("Digite o salario atual: ");
    scanf("%f", &sl);
    if (sl > 0 && sl <= 500) {
        ida = sl * 0.15;
    }
    else if (sl >= 501 && sl <= 700) {
        ida = sl * 0.12;
    }
    else if (sl >= 701 && sl <= 1000) {
        ida = sl * 0.10;
    }
    else if (sl >= 1001 && sl <= 1800) {
        ida = sl * 0.07;
    }
    else if (sl >= 1801 && sl <= 2500) {
        ida = sl * 0.04;
    }
    else if (sl >= 2500) {
        ida = 0;
    }
    else {
        printf("Por favor, digite um valor valido\n");
        return 1; 
    }
    nvs = sl + ida;
    printf("\nNome do usuario: %s\n", nome);
    printf("Percentual de aumento: %.2f%%\n", (ida / sl) * 100);
    printf("Salario atual: R$%.2f\n", sl);
    printf("Novo salario: R$%.2f\n", nvs);
    system("pause");
    return 0;
}
